src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -963,7 +963,7 @@ MavlinkReceiver::handle_message_vision_position_estimate(mavlink_message_t *msg)
 	// Use the component ID to identify the vision sensor
 	vision_position.id = msg->compid;
 
-	vision_position.timestamp_boot = hrt_absolute_time(); // Monotonic time
+	vision_position.timestamp_boot = sync_stamp(pos.usec);
```

Comment:
What is the idea behind using `.timestamp_computer`? Why don't we just save the synced timestamp to `.timestamp_boot`. If time sync works than we want to use the received timestamps, if time sync does not work than `sync_stamp()` returns `hrt_absolute_time()` anyway.